docs/support.md

```diff
@@ -0,0 +1,213 @@
+(support-features)=
+(support-utilities)=
+# Support Features
+
+The package bundles a few support and utility functions that try to fill a few
+gaps you will observe when working with CrateDB, a distributed OLAP database,
+since it lacks certain features, usually found in traditional OLTP databases.
+
+A few of the features outlined below are referred to as [polyfills], and
+emulate a few functionalities, for example, to satisfy compatibility issues on
+downstream frameworks or test suites. You can use them at your disposal, but
+you should know what you are doing, as some of them can seriously impact the
+performance.
+
+Other features include efficiency support utilities for 3rd-party frameworks,
+which can be used to increase performance, mostly on INSERT operations.
+
+
+(support-insert-bulk)=
+## Bulk Support for pandas and Dask
+
+:::{rubric} Background
+:::
+CrateDB's [](inv:crate-reference#http-bulk-ops) interface enables efficient
+INSERT, UPDATE, and DELETE operations for batches of data. It allows to issue
```

Comment:
```suggestion
INSERT, UPDATE, and DELETE operations for batches of data. It enables
```

Changed to read "It enables bulk operations".

Reply:
Thank you!